### ISSUE TITLE
New Styles

### ISSUE THREAD

**Issue by jfellien** [misgod/fast-neural-style-keras#1]
Hi,
this is such a great project you have created. Your pretrained styles are awesome but after a while of playing around I want to add my own styles. How can I do this?

BR
~me

**Comment by misgod** [comment]
Using train.py to train your own style, and something in train.py you should note.

#51 train_batchsize=1 
I found larger than 1 will make network un-stable to produce quality image.(It seems to be related to  batch norm implement )

#52 train_image_path = "images/train/" 
mscoco dataset should be here

#33 return "images/style/"+style+".jpg"  
Your style image, and you should pass the style name with -s argument when you start training.(see main())

#20 display_img()
This will produce intermediate images while training for you evaluate.

**Comment by jfellien** [comment]
Hey, thank you so far. But I changed anything you described but now I get an error `ImportError: cannot Import Name _obtain_input_shape`


I'm very new in all of the ML an TF and Keras and Python and so on ;-)
I have not really an idea what I do. So Maybe you can help me a bit.

**Comment by stgstg27** [comment]
This issue occured because of the version of keras.

In my case, I was downgrade keras 2.2.2 to 2.2.0, and the problem was solved.

**Comment by juggernaut09** [comment]
Hi,
great implementation , I wanted to try my own custom style , in order to do so , I got confused how to train it .As you mention here to download MSCOCO dataset ,the size of the data set is large and the other doubt is once the data set is downloaded can we just plug it there and run the train.py file ?

You have used dummy_y as output to train #line61 in train.py, can you please explain it.
Can you please help me

**Comment by misgod** [comment]
Yes, MSCOCO dataset size is 20G, and you can start to train your own model after downloaded.

The dummy_y is for #84 model.train_on_batch(x, dummy_y) because you have to pass y to call train function.
You also can find dummy loss #57 model.compile(optimizer, dummy_loss).
Our losses are calculated from feature maps of specific layers in pre-trained VGG not from output.
So the tricky part is to use regularizes as loss to achieve that.
